### PR TITLE
Add KUBECONFIG to .profile and .bashrc | Restructure Nvidia container

### PR DESCRIPTION
**Pull request by michael-chi**
1. Add `KUBECONFIG` environment variable to `.profile` and `.bashrc`
2. Update NVIDIA device plugin installation URL
3. Move NVIDIA container related scripts to be executed in separate script after ABM is installed.

### CODE REVIEW

File: edge-server/anthos/node-setup-common.sh.tmpl [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env sh
 
+KUBECONFIG="$SETUP_DIR/bmctl-workspace/$ANTHOS_MEMBERSHIP_NAME/$ANTHOS_MEMBERSHIP_NAME-kubeconfig"
```
Comment (ferrarimarco) [inline edge-server/anthos/node-setup-common.sh.tmpl:3]: Can you add `{` and `}` to delimit vars?

File: edge-server/anthos/node-setup-common.sh.tmpl [hunk 2]
```diff
@@ -137,91 +143,29 @@ setup_vlan_control_plane() {
...
+    USER_HOME_PATH="${1}"
+    echo "[Info] USER_HOME_PATH=${USER_HOME_PATH}"
+    shift
   fi
+  echo "[Info] KUBECONFIG=${KUBECONFIG}"
+  echo "[Info] Appending KUBECONFIG to ${USER_HOME_PATH}/.bashrc"
+  echo "export KUBECONFIG=${KUBECONFIG}" >>"${USER_HOME_PATH}/.bashrc"
+
+  echo "[Info] Appending KUBECONFIG to ${USER_HOME_PATH}/.profile"
+  echo "export KUBECONFIG=${KUBECONFIG}" >>"${USER_HOME_PATH}/.profile"
```
Comment (ferrarimarco) [inline edge-server/anthos/node-setup-common.sh.tmpl:163]: Avoid repeating paths using environment variables.

Comment (ferrarimarco) [inline edge-server/anthos/node-setup-common.sh.tmpl:163]: What if `.bashrc` and `.profile` don't exist?

**Review by ferrarimarco — CHANGES_REQUESTED** [review]

File: edge-server/anthos/node-setup-common.sh.tmpl (hunk 1 above)
Comment (michael-chi) [reply edge-server/anthos/node-setup-common.sh.tmpl:3]: updated

File: scripts/common.sh [hunk 3]
```diff
@@ -49,8 +49,12 @@ ERR_UNSUPPORTED_OS_DESCRIPTION="[Error]Unsupported OS"
...
 # shellcheck disable=SC2034
 GCLOUD_CLI_CONTAINER_IMAGE_ID="gcr.io/google.com/cloudsdktool/cloud-sdk:397.0.0"
-# shellcheck disable=SC2034
-TERRAFORM_CONTAINER_IMAGE_ID="$(grep <docker/terraform/Dockerfile "hashicorp/terraform" | awk -F ' ' '{print $2}')"
+if [ -f "docker/terraform/Dockerfile" ]; then
+  # shellcheck disable=SC2034
+  TERRAFORM_CONTAINER_IMAGE_ID="$(grep <docker/terraform/Dockerfile "hashicorp/terraform" | awk -F ' ' '{print $2}')"
+else
+  echo "[Warning] docker/terraform/Dockerfile does not exist. Variable TERRAFORM_CONTAINER_IMAGE_ID not set."
+fi
```
[the remark this answers is not in the document]
Comment (michael-chi) [reply scripts/common.sh:57]: updated.

File: edge-server/anthos/node-setup-common.sh.tmpl (hunk 2 above)
Comment (michael-chi) [reply edge-server/anthos/node-setup-common.sh.tmpl:163]: Add a variable for file path.

Comment (michael-chi) [reply edge-server/anthos/node-setup-common.sh.tmpl:163]: They are created when a new user is created. The script is run after OS installation completed where we have user`ubuntu` and user `viai-admin` created. When the script is ran they should already be there.

The script either:

- List folders under `/home`, and iterate through each folders to create / append the file. Or
- Update files in '/home/viai-admin', so there is a chance somehow `viai-admin` was not created.
   * Added a check here to confirm wether the folder exists.

If the file was somehow not created, `>>` automatically create the file, so we are still good.

**Review by ferrarimarco — APPROVED** [review]

**Pull request merged**